assets/src/dashboard/icons/stories/index.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+/*eslint import/namespace: ['error', { allowComputed: true }]*/
```

Comment:
Neat! Although `eslint-disable-next-line` would probably work just as well without affecting the rest of the file I think

Reply:
Actually, I remember why i did it this way last week - eslint still thinks there's an error since it's breaking a rule on how the imports are getting in here: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/namespace.md 

I need to actually tell eslint import/namespace to allowComputed on this file otherwise it won't accept it even if it's getting ignored. I _can_ update it from 'error' to be 2 for depth. 

If you know another way around this let me know, this is what I came up with reading the docs for this rule.

Reply:
I think you might not need the `eslint` comment you have on import, if you use a disable-next-line comment right before you access `Icons[iconName]`, like this image shows:

![image](https://user-images.githubusercontent.com/40646372/82934573-879a9680-9f40-11ea-9408-171de38657d7.png)

Reply:
you're right. thank you!